SocketRocket/SocketRocket/Internal/Proxy/ARTSRProxyConnect.m

```diff
@@ -287,7 +287,28 @@ - (void)_runPACScript:(NSString *)script withProxySettings:(NSDictionary *)proxy
         NSString *proxyType = settings[(NSString *)kCFProxyTypeKey];
         [self _readProxySettingWithType:proxyType settings:settings];
     }
-    [self _openConnection];
+    [self openConnection];
+}
+
+- (void)openConnection
```

Comment:
Since it was considered as a temporary fix until completely removal of this API in another PR, I haven't put much of an effort in naming here, I've just made sure that all call sites of `_openConnection` function now call the new method, which is just a "conditional wrapper" over the old one.

Reply:
Resolving this one since @QuintinWillison mentioned on Slack he'd prefer to focus on getting this released.